src/main/java/org/apache/ibatis/session/TransactionIsolationLevel.java

```diff
@@ -25,7 +25,12 @@
   READ_COMMITTED(Connection.TRANSACTION_READ_COMMITTED),
   READ_UNCOMMITTED(Connection.TRANSACTION_READ_UNCOMMITTED),
   REPEATABLE_READ(Connection.TRANSACTION_REPEATABLE_READ),
-  SERIALIZABLE(Connection.TRANSACTION_SERIALIZABLE);
+  SERIALIZABLE(Connection.TRANSACTION_SERIALIZABLE),
+  /**
+  * A non-standard isolation level for Microsoft SQL Server. Defined in
+  * the SQL Server JDBC driver {@link com.microsoft.sqlserver.jdbc.ISQLServerConnection}
+  */
```

Comment:
Please specify `@since 3.5.6`.

Reply:
Thank you, I missed this one (as usual). It's done.